ArduCopter/RC_Channel.cpp

```diff
@@ -432,6 +433,7 @@ bool RC_Channel_Copter::do_aux_function(const AUX_FUNC ch_option, const AuxSwitc
                 break;
             case AuxSwitchPos::LOW:
                 copter.attitude_control->set_inverted_flight(false);
+                copter.force_flying = false;
```

Comment:
Setting here misses the case where we revert inverted flight on mode change.

I think its better to add a helper function for the force flying flag.

Something like:
```
bool Copter::get_force_flying() const
{
#if FRAME_CONFIG == HELI_FRAME
    if (attitude_control.get_inverted_flight()) {
        return true;
    }
#endif
    return force_flying;
}
```
Then you would call the helper in where `force_flying` is accessed directly now.

Reply:
Implemented. thanks!